pvmismatch/contrib/ModuleMismatchSimulator.py

```diff
@@ -0,0 +1,510 @@
+'''
```

Comment:
Please add a shebang so I don't have to type `python ModuleMismatchSimulatory.py` to start it.